weblate/accounts/admin.py

```diff
@@ -92,6 +94,29 @@ def user_groups(self, obj):
         """
         return ','.join([g.name for g in obj.groups.all()])
 
+    def __init__(self, *args, **kwargs):
+        super(WeblateUserAdmin, self).__init__(*args, **kwargs)
+
+        groups = Group.objects.all()
```

Comment:
I didn't expected that admin __init__ is called before database. I need run this code when the database is ready but before admin page is rendered. Do you have idea how to arrange it?